useful-scripts/misc-test-utils/src/services/commonService.ts

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import chalk from 'chalk';
+import StreamZip from 'node-stream-zip';
+import globby from 'globby';
+import fs from 'fs';
+import rimraf from 'rimraf';
+import { projectRoot } from '../config';
+import errorHandler from '../utils/errorHandler';
+import { ReleaseData } from '../types/ReleaseData';
+import sleep from '../utils/sleep';
+
+const { ADMIN_URL, JWT_TOKEN } = process.env;
+
+const cwd = projectRoot;
+
+const commonService = {
+  validate: async () => {
```

Comment:
`validate` is a bit of a weird name as it's mostly checking that directories exist. Would suggest changing it to something else like `prepareDirectories`